core/modules/color/color.module

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
Can we remove this extra space? (and maybe if your editor added it - you can configure it not to, as we don't like these extra lines in Backdrop)

Reply:
I think if that's a big enough deal to hold up a PR, it should be documented here: https://api.backdropcms.org/php-standards#indenting as it is on d.o.

Reply:
Just took a look https://www.drupal.org/docs/develop/standards/coding-standards#indenting - I guess Drupal does like the blank line.

Reply:
I think it should be documented even if it's not a big enough deal to hold up a pr, so https://api.backdropcms.org/php-standards :)